js_course/test-2.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('test-2', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.locator('[data-test="username"]').click();
```

Comment:
We can move this to fixture

```
  await page.locator('[data-test="username"]').click();
  await page.locator('[data-test="username"]').fill('standard_user');
  await page.locator('[data-test="password"]').click();
  await page.locator('[data-test="password"]').fill('secret_sauce');
  await page.locator('[data-test="login-button"]').click();
```

Doc - https://playwright.dev/docs/test-fixtures

Reply:
@divchunonka let's ignore this comment for now. We'll come back to it in the next task.